scripts/download_cams_pm25.py

```
@@ -0,0 +1,267 @@
+            sensors = await datasource.get_sensors(location, parameters=param_types)
+            
+            for sensor in sensors:
+                # Get measurements
+                measurement_count = 0
+                async for measurements in datasource.get_measurements(
+                    sensor,
+                    start_date=start_date,
+                    end_date=end_date
+                ):
+                    # Write measurements to CSV
+                    with open(output_file, 'a') as f:
```

Ruff (0.12.2)

Async functions should not open files with blocking methods like `open`

(ASYNC230)